rules/opentitan.bzl

```diff
@@ -389,41 +640,55 @@ def _unique_deps(*deplists):
 
 def opentitan_functest(
         name,
-        platform = OPENTITAN_PLATFORM,
         targets = ["verilator", "cw310"],
         args = [],
         data = [],
         ottf = _OTTF_DEPS,
+        test_in_rom = False,
+        signed = False,
+        key = "test_key_0",
         verilator = None,
         cw310 = None,
         **kwargs):
     """A helper macro for generating OpenTitan functional tests.
-    This macro is mostly a wrapper around opentitan_binary, but creates
-    testing artifacts for each of the keys in `per_device_deps`.
-    The testing artifacts are then given to an `sh_test` rule which
-    dispatches the test via opentitantool.
+
+    This macro is mostly a wrapper around opentitan_flash_binary, but creates
+    testing artifacts for each of the hardware targets in `targets`. The testing
+    artifacts are then given to an `sh_test` rule which dispatches the test to
+    the corresponding hardware target via opentitantool.
     Args:
       @param name: The name of this rule.
-      @param platform: The target platform for the artifacts.
-      @param targets: A list of targets on which to dispatch tests.
+      @param targets: A list of hardware targets on which to dispatch tests.
       @param args: Extra arguments to pass to `opentitantool`.
       @param data: Extra data dependencies needed while executing the test.
-      @param ottf: Default dependencies for OTTF tests.  Set to empty list if
+      @param ottf: Default dependencies for OTTF tests. Set to empty list if
                    your test doesn't use the OTTF.
-      @param **kwargs: Arguments to forward to `opentitan_binary`.
+      @param test_in_rom: Whether or not the test will run out of ROM. If false,
+                          test defaults to running out of flash.
```

Comment:
nit: It defaults to running out of flash if test_in_rom is undefined. It runs out of flash if it's undefined or false. 

```
 @param test_in_rom: Whether to run the test from ROM, Runs from flash by default.

```

Reply:
Fixed.